cheta/comps/ephem_stk.py

```diff
@@ -0,0 +1,276 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""
+STK orbit ephemeris computed MSIDs
+"""
+
+import calendar
+import functools
+import logging
+import re
+from pathlib import Path
+
+import numpy as np
+from astropy.table import Column, Table
+from cxotime import CxoTime, CxoTimeLike
+from kadi import occweb
+
+from .computed_msid import ComputedMsid
+
+logger = logging.getLogger("cheta.fetch")
+
+
+EPHEM_STK_RECENT_DIR = "FOT/mission_planning/Backstop/Ephemeris"
+EPHEM_STK_ARCHIVE_DIR = "FOT/mission_planning/Backstop/Ephemeris/ArchiveMCC"
+
+MONTH_NAME_TO_NUM = {
+    mon: f"{ii + 1:02d}" for ii, mon in enumerate(calendar.month_abbr[1:])
+}
+
+
+def read_stk_file(path, format="stk", **kwargs):
+    """Read a STK ephemeris file from OCCweb and return an astropy table.
+
+    For format "stk" the table is the same as the file with columns:
+
+           name     dtype
+      ----------- -------
+      Time (UTCG)   str23
+           x (km) float64
+           y (km) float64
+           z (km) float64
+      vx (km/sec) float64
```

Comment:
For even the stk format, since we don't have users yet, could we go with the "x", "y", "z" etc column names and properly put the units in the astropy table?  Though I suppose I'm not sure about appropriate units for the Time field.

Reply:
I just don't think this pathway will ever get even one user, so it isn't clear if spending time is worth it. I would probably also change the `format` default to `"cxc"`.